client/client-multi/client-cpp/Modules/FindAvro.cmake

```diff
@@ -61,7 +61,7 @@ find_path(AVRO_INCLUDE_DIR
      ${CMAKE_INCLUDE_PATH}
 )
 
-if (AVRO_LINK_STATIC)
+if (Avro_USE_STATIC_LIBS)
     set (AVRO_LOOK_FOR_LIB_NAMES avrocpp_s avrocpp)
```

Comment:
I would exclude `avrocpp` from here as we can accidentally link with shared version.